### PR TITLE
Examples: Update Servo Syntax

### PR DESCRIPTION
**Pull request by dtex**

### CODE REVIEW

File: eg/servo-drive.js [hunk 1]
```diff
@@ -26,11 +25,11 @@ board.on("ready", function() {
     invert: true // one wheel mounted inverted of the other
   });
 
-  wheels.both = new five.Servos().stop(); // reference both together
+  wheels.both = new Servos().stop(); // reference both together
```
Comment (rwaldron) [inline eg/servo-drive.js]: One thing I will never forgive myself for: allowing these to implicitly find all instances of any Servo. Can you update this to: 

```js
wheels.both = new Servos([wheels.left, wheels.right]).stop();
```

File: eg/servo-drive.js [hunk 2]
```diff
@@ -39,8 +38,6 @@ board.on("ready", function() {
   wheels.both.cw();
 
   // Stop driving after 3 seconds
-  this.wait(3000, function() {
-    wheels.both.stop();
-  });
+  board.wait(3000, wheels.both.stop);
```
Comment (rwaldron) [inline eg/servo-drive.js]: This should be safe because `wait` does not bind `this`

File: eg/servo-slider.js [hunk 3]
```diff
@@ -1,14 +1,12 @@
...
-  var tilt = new five.Servo(9);
+  const slider = new Sensor("A0");
+  const tilt = new Servo(9);
 
-  slider.scale([0, 180]).on("slide", function() {
-
-    // The slider's value will be scaled to match the tilt servo range
-    tilt.to(this.value);
+  slider.on("slide", () => {
+    tilt.to((slider.scaleTo(0, 180)));
```
Comment (rwaldron) [inline eg/servo-slider.js]: There's a set of unnecessary parens in there. Have you been writing C?

Comment (dtex) [reply eg/servo-slider.js]: I'm sure it was my editor "helping"

File: eg/servo-PCA9685.js [hunk 4]
```diff
@@ -1,25 +1,24 @@
-var five = require("../lib/johnny-five.js");
-var board = new five.Board();
+const {Board, Servo} = require("../lib/johnny-five.js");
+const board = new Board();
 
-board.on("ready", function() {
+board.on("ready", () => {
   console.log("Connected");
```
[the remark this answers is not in the document]
Comment (dtex) [reply eg/servo-PCA9685.js]: Done

File: eg/servo-PCA9685.js [hunk 5]
```diff
@@ -1,25 +1,24 @@
...
-  var a = new five.Servo({
+  const a = new Servo({
     address: 0x40,
     controller: "PCA9685",
     pin: 0,
   });
 
-  var b = new five.Servo({
+  const b = new Servo({
     address: 0x40,
```
[the remark this answers is not in the document]
Comment (dtex) [reply eg/servo-PCA9685.js]: Done

File: eg/servo-animation-leg.js [hunk 6]
```diff
@@ -57,13 +57,11 @@ var board = new five.Board().on("ready", function() {
...
    */
-  var stand = {
+  const stand = {
     duration: 500,
     loop: false,
     cuePoints: [0, 0.1, 0.3, 0.7, 1.0],
-    oncomplete: function() {
-      ph.state = "stand";
-    },
+    oncomplete: () => ph.state = "stand",
```
[the remark this answers is not in the document]
Comment (dtex) [reply eg/servo-animation-leg.js]: Done

File: eg/servo-diagnostic.js [hunk 7]
```diff
@@ -18,13 +17,12 @@ var five = require("../lib/johnny-five.js"),
...
 
-pins = [];
-ranges = [];
-
-args.forEach(function(val) {
-  var vals = val.split(":").map(function(v) {
+args.forEach(val => {
+  const vals = val.split(":").map(v => {
     return +v;
   });
```
[the remark this answers is not in the document]
Comment (dtex) [reply eg/servo-diagnostic.js]: Done

File: eg/servo-diagnostic.js [hunk 8]
```diff
@@ -36,23 +34,23 @@ args.forEach(function(val) {
...
 
-  servos = new five.Servos();
+  const servos = new Servos();
 
   servos.center();
 
   // Inject a Servo Array into the REPL as "s"
-  this.repl.inject({
+  board.repl.inject({
     s: servos
```
[the remark this answers is not in the document]
Comment (dtex) [reply eg/servo-diagnostic.js]: Done

File: eg/servo-drive.js (hunk 1 above)
Comment (dtex) [reply eg/servo-drive.js]: Ya, that's weird. Done.

File: eg/servo-drive.js (hunk 2 above)
Comment (dtex) [reply eg/servo-drive.js]: Ah, now it's clear. TY

**Comment by rwaldron** [comment]
I think that my review was still active in another tab and I may have submitted it twice??

File: docs/servo-prompt.md [hunk 9]
```diff
@@ -31,26 +31,26 @@ node eg/servo-prompt.js
...
+const {Board, Servo} = require("johnny-five");
+const readline = require("readline");
 
-var rl = readline.createInterface({
+const rl = readline.createInterface({
   input: process.stdin,
   output: process.stdout
 });
 
-five.Board().on("ready", function() {
```
Comment (rwaldron) [inline docs/servo-prompt.md]: Woof. Bye-bye icky practices

**Pull request merged**